packages/calcite-components/src/components/text-area/text-area.tsx

```diff
@@ -161,7 +161,7 @@ export class TextArea
   @property() label: string;
 
   /**
-   * Specifies the maximum number of characters allowed.
+   * Specifies the maximum number of characters allowed. Must be greater than or equal to the value of `minlength`, if present and valid.
```

Comment:
This should be `minLength` since we're documenting the property. Applies to `maxLength`'s doc.